operator/src/main/java/oracle/kubernetes/operator/helpers/CrdHelper.java

```diff
@@ -616,6 +641,15 @@ public NextAction onSuccess(
         LOGGER.info(MessageKeys.CREATING_CRD, callResponse);
         return doNext(packet);
       }
+
+      @Override
+      protected NextAction onFailureNoRetry(Packet packet, CallResponse<V1beta1CustomResourceDefinition> callResponse) {
```

Comment:
New functionality without a unit test

Reply:
I think this is covered by whenBetaCrdReplaceFails_scheduleRetryAndLogFailedMessageInOnFailureNoRetry() unit test. Please let me know if more tests needs to be added.

Reply:
I have added a new unit test whenBetaCrdReplaceThrowsStreamResetException_scheduleRetryAndLogFailedMessageInOnFailureNoRetry() for StreamResetException check.